.github/workflows/linux-qe-template.yml

```diff
@@ -0,0 +1,200 @@
+          eval $(ssh-agent -s)
+          echo $SSH_AUTH_SOCK > ssh_auth_sock
+          echo $SSH_AGENT_PID > ssh_agent_pid
+          ssh-add id_rsa
+
+      - name: reserve machine from testing farm
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        env: 
+          TESTING_FARM_API_TOKEN: ${{ secrets.TESTING_FARM_API_TOKEN }}
+          PULL_SECRET: ${{ secrets.PULL_SECRET }}
+        run: |
+          echo "${PULL_SECRET}" > pull-secret
+          export TESTING_FARM_API_TOKEN=${TESTING_FARM_API_TOKEN}
+          testing-farm reserve --compose Fedora-40 --duration 240 --arch aarch64 --ssh-public-key id_rsa.pub --no-autoconnect | tee info
+          machine=`tail -n 1 info`
+          echo ${machine##*@} > host
+          echo root > username
+          echo proxy > bastion_username
+          echo testing-farm.io > bastion_host
+     
+      - name: Download linux binary
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        uses: actions/download-artifact@v4
+        with:
+          name: linux-binary
+          run-id: ${{inputs.trigger-workflow-run-id}}
+          github-token: ${{ github.token }}
+
+      - name: Download qe oci image
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        id: download-qe-oci-image-artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: crc-${{inputs.qe-type}}-linux-arm64
+          run-id: ${{inputs.trigger-workflow-run-id}}
+          github-token: ${{ github.token }}
+
+      - name: Download integration qe oci image
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        id: download-qe-oci-image-artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: crc-integration-linux-arm64
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ github.token }}
+
+      - name: Install CRC on host
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        run: |
+          podman run --rm -d --privileged --name crc-linux-install-${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}} \
+            -e TARGET_HOST=$(cat host) \
+            -e TARGET_HOST_USERNAME=$(cat username) \
+            -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+            -e BASTION_HOST_USERNAME=$(cat bastion_username) \
+            -e BASTION_HOST=$(cat bastion_host) \
+            -e TARGET_FOLDER=crc-support \
+            -e TARGET_CLEANUP='false' \
+            -e OUTPUT_FOLDER=/data \
+            -e DEBUG='true' \
+            -e SSH_AUTH_SOCK=$(cat ssh_auth_sock) \
+            -v "$(cat ssh_auth_sock):$(cat ssh_auth_sock)" \
+            -v ${PWD}:/data:z \
+            -v ${PWD}/crc:/opt/crc-support/crc:z \
+            quay.io/rhqp/crc-support:v0.5.1-linux crc-support/install.sh crc-support/crc
```

Comment:
`quay.io/crc-org/ci-crc-support:v1.0.0-linux` Is now available. If possible change to this as from now on it should be the "official" one.

Let me do a check on using it directly with `crc` as a local asset. And will add here the snippet to make use of it here.